iceoryx_examples/icediscovery/README.md

```diff
@@ -47,7 +47,7 @@ It is included via:
 On that object we can call the method `findService` which expects the three
```

Comment:
What do you think about commenting out the expected output section since the asciicast is not in sync with your changes?

Reply:
Good point, I'll add a todo to re-record it as we've done it before.